frontend/src/InteractiveImport/Interactive/InteractiveImportRow.js

```diff
@@ -82,7 +83,7 @@ class InteractiveImportRow extends Component {
     const isValid = !!(
       artist &&
       album &&
-      tracks.length &&
+      (isSingleFileRelease || tracks.length) &&
```

Comment:
Shouldn't `tracks.length` apply regardless?

Reply:
I might have encountered some issues before so added it like this. Probably just some false positives. I'll revert this part.